includes/import/class-wc-product-csv-importer.php

```diff
@@ -69,7 +69,9 @@ protected function read_file() {
 				fseek( $handle, (int) $this->params['start_pos'] );
 			}
 
-			while ( false !== ( $row = fgetcsv( $handle, 0, $this->params['delimiter'], $this->params['enclosure'] ) ) ) {
+			$row = fgetcsv( $handle, 0, $this->params['delimiter'], $this->params['enclosure'] );
+
+			while ( false !== $row ) {
```

Comment:
We need `$row = fgetcsv( $handle, 0, $this->params['delimiter'], $this->params['enclosure'] );` inside the loop, also. Otherwise it will just loop over the same row over and over again:
<img width="401" alt="screen shot 2018-01-09 at 9 12 56 am" src="https://user-images.githubusercontent.com/7317227/34733516-8477f026-f51d-11e7-8b5f-22da5add8359.png">